Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md

```diff
@@ -0,0 +1,446 @@
+
+- Many countries use the dataset and work well in real-Globe scenarios. For each data set, information about content and licenses are provided. Proper permissions have been cited for all the datasets, and we urge you to follow the license for every dataset before proceeding to work with it.
+
+If you have another dataset on your mind that would be beneficial to [Moja Global's mission](https://moja.global/), please feel free to reach out to us on [info@moja.global](info@moja.global) to share details about this dataset.
+
+---
+
+### 1.0.1 Data Analysis Software and Notebooks
+
+Select software that supports geospatial analysis. 
+- Moja Global is an open-source project, and the interest here is to share your process (code) and results (explanation) to support the open-source movement. In order to effectively record your analysis, we recommend that you use a [notebook.](https://sourceforge.net/software/product/Google-Colab/alternatives). 
+
+
+---
+
+### 1.0.2 Contributing
+
+
+When your contribution is ready; raise a PR to have it merged. Please refer to the [community website](https://community.moja.global/community/code-contribution) to know how to make a pull request.
+
+>If you have experience and want to help, please send an email to [mentorship@moja.global](mailto:mentorship@moja.global). We are open to mentoring new developers and contributors through various mentorship program and initiatives.
+
+---
+
+### 1.0.3 The Analysis
+
+When we look at climate action, we would like to get descriptions from a global scale to a specific state administration within a particular country. A draft review of the characteristics of each file in the dataset is found [here](https://github.com/Simpleshell3/Land-Sector-Dataset-Description/blob/main/Land-Sector-Dataset-Description.md).
+
+Amongst the numerous factors that affecct climate change, the Agriculture, Forestry and Other Land Use (AFOLU) sector is responsible for almost a quarter of the global Greenhouse gases (GHG) emissions. The emissions associated with AFOLU activities are projected to increase in the future.
+
+
+The following sections of this guide will direct you on how to document your analysis at any scale(global, continental, country, state).
+
+[**1.1 Global Analysis**](https://github.com/Simpleshell3/About_moja_global/blob/master/Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md#11-global-analysis)
+
+[**1.2 Continental Analysis**](https://github.com/Simpleshell3/About_moja_global/blob/master/Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md#12-continental-analysis)
+
+[**1.3 Country Analysis**](https://github.com/Simpleshell3/About_moja_global/blob/master/Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md#13-Country-analysis)
+
+[**1.4 State/Province/County/Region/City Analysis**](https://github.com/Simpleshell3/About_moja_global/blob/master/Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md#14--stateprovincecountyregioncity-analysis)

```

Comment:
The hyperlinks should be relative and not be directly referenced through links. If you delete your personal fork, the access to them would be lost.